Source/WebCore/svg/SVGFEColorMatrixElement.cpp

```diff
@@ -127,9 +127,9 @@ RefPtr<FilterEffect> SVGFEColorMatrixElement::createFilterEffect(const FilterEff
         switch (filterType) {
         case FECOLORMATRIX_TYPE_MATRIX: {
             static constexpr unsigned matrixValueCount = 20;
-            filterValues.reserveInitialCapacity(matrixValueCount);
-            for (size_t i = 0; i < matrixValueCount; i++)
-                filterValues.uncheckedAppend((i % 6) ? 0 : 1);
+            filterValues = Vector<float>(matrixValueCount, [](size_t i) {
+                return (i % 6) ? 0. : 1.;
```

Comment:
I know `0.` and `1.` work, but `0.0` and `1.0` are more traditional, I think. Or we could use `-> double`.

Reply:
Oops. I had missed this review comment. Will fix and next iteration.